pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -493,6 +461,50 @@ func (r *ReconcileIBMBlockCSI) reconcileServiceAccount(instance *ibmblockcsi.IBM
 			if err != nil {
 				return err
 			}
+
+			controllerStatefulset, err := r.getControllerStatefulSet(instance)
+			if err != nil {
+				return err
+			}
+
+			nodeDaemonSet, err := r.getNodeDaemonSet(instance)
+			if err != nil {
+				return err
+			}
+
+			if controllerServiceAccountName == sa.Name {
+				controllerlogger := log.WithValues("Resource Type", "Controller")
+				controllerPod := &corev1.Pod{}
+				err := r.getControllerPod(controllerStatefulset, controllerPod)
+				if err != nil {
```

Comment:
what about `errors.IsNotFound(err)`? what if the driver just wasn't deployed yet?

Reply:
done